workbook.go

```diff
@@ -123,6 +123,11 @@ func (o CodeName) setWorkbookPrOption(pr *xlsxWorkbookPr) {
 	pr.CodeName = string(o)
 }
 
+// setWorkbookPrOption implements the WorkbookPrOption interface.
```

Comment:
I suggest the get and set implement functions following the order in [OOXML Spec](https://github.com/sc34wg4/OOXMLSchemas/blob/5d7db375ff5f5ecfd38d39d088a6b81a6c41e39e/Part1/OfficeOpenXML-XMLSchema-Strict/sml.xsd#L4200-L4220), move this function before `CodeName`, and update the available options document of the functions `SetWorkbookPrOptions` and `GetWorkbookPrOptions`